pkg/parse/namespace.go

```diff
@@ -88,11 +88,19 @@ func (p *namespace) parseSource(ctx context.Context, state sourceState) ([]ast.F
 		// So disable the API call and dynamic mode of NamespaceSelector.
 		AllowAPICall:             false,
 		DynamicNSSelectorEnabled: false,
+		WebhookEnabled:           p.WebhookEnabled,
 	}
 	options = OptionsForScope(options, p.Scope)
 
 	objs, err = validate.Unstructured(ctx, p.Client, objs, options)
 
+	if !p.WebhookEnabled {
+		klog.Infof("Removing %s annotation as Admission Webhook is disabled", metadata.DeclaredFieldsKey)
```

Comment:
I would suggest making the log at level 3. Otherwise, the log would show up for every sync when the webhook is disabled.

Reply:
Just realized this segment is repeated in namespace.go, root.go and run.go, cleaned up so it's done in run.go once. Fixed log level.